Source/Data/SpectrumAnalysis/KTSequentialLineData.hh

```diff
@@ -0,0 +1,93 @@
+/*
+ * KTSequentialLineData.hh
+ *
+ *  Created on: Sep 8, 2016
+ *      Author: Christine
+ */
+
+#ifndef KTSEQUENTIALLINEDATA_HH_
+#define KTSEQUENTIALLINEDATA_HH_
+
+#include "KTMemberVariable.hh"
+#include "KTData.hh"
+#include "KTDiscriminatedPoint.hh"
+
+#include <string>
+#include <vector>
+#include <set>
+
+namespace Katydid
+{
+    class KTSequentialLineData : public Nymph::KTExtensibleData< KTSequentialLineData >
+    {
+
+        private:
+
+            KTDiscriminatedPoints fLinePoints;
+            std::vector<double> fAmplitudeList;
+            std::vector<double> fSNRList;
+
+            //Points fPoints;
+
+        public:
+
+            static const std::string sName;
+
+            MEMBERVARIABLE(double, StartTimeInRunC);
+            MEMBERVARIABLE(double, EndTimeInRunC);
+            MEMBERVARIABLE(double, StartTimeInAcq);
+            MEMBERVARIABLE(double, EndTimeInAcq);
+            MEMBERVARIABLE(double, StartFrequency);
+            MEMBERVARIABLE(double, EndFrequency);
+            MEMBERVARIABLE(double, InitialSlope);
+            MEMBERVARIABLE(double, Slope);
+            MEMBERVARIABLE(unsigned, Component);
+            MEMBERVARIABLE(unsigned, CandidateID)
+            MEMBERVARIABLE(double, AmplitudeSum);
+            MEMBERVARIABLE(double, SNRSum);
+            MEMBERVARIABLE(double, NUPSum);
+            MEMBERVARIABLE(unsigned, NPoints);
+            MEMBERVARIABLE(uint64_t, AcquisitionID);
+            MEMBERVARIABLE(unsigned, MinPoints);
+            MEMBERVARIABLE(double, StartFrequencySigma);
+            MEMBERVARIABLE(double, EndFrequencySigma);
+
+            double fSumX;
+            double fSumY;
+            double fSumXY;
+            double fSumXX;
+
+            KTSequentialLineData();
```

Comment:
Public variables again...

Reply:
I second Mathieu's comment.  The variables should be private with any necessary access functions provided.  Feel free to use a MEMBERVARIABLE macro if appropriate.